temperatures/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DbTemperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime, nullable=False)
+    temperature = Column(Float, nullable=False)
+    city_id = Column(Integer, ForeignKey("city_id"))
```

Comment:
The ForeignKey reference should specify the table name and column name in the format 'table_name.column_name'. It seems like 'city_id' is being used as the table name, which is incorrect. Please ensure that the correct table name is used here.